apps/api/src/app/widgets/widgets.controller.ts

```diff
@@ -228,6 +231,25 @@ export class WidgetsController {
     return await this.removeMessageUsecase.execute(command);
   }
 
+  @ApiOperation({
+    summary: `Remove a subscriber's feed messages`,
+  })
+  @UseGuards(AuthGuard('subscriberJwt'))
+  @Delete('/messages')
+  async removeAllMessages(
+    @SubscriberSession() subscriberSession: SubscriberEntity,
+    @Query('feedId') feedId?: string
+  ): Promise<number> {
```

Comment:
do we need to return how many were deleted?

Reply:
no not necessary, Shall I return boolean?